agent/session_endpoint_test.go

```diff
@@ -715,9 +717,14 @@ func TestSessionsForNode(t *testing.T) {
 		if !ok {
 			t.Fatalf("should work")
 		}
-		if len(respObj) != 10 {
-			t.Fatalf("bad: %v", respObj)
+		// TODO: this assertion would be much cleaner with go-cmp.Equals and cmpopts.SortSlices
+		respIDs := make([]string, 0, len(ids))
+		for _, session := range respObj {
+			respIDs = append(respIDs, session.ID)
 		}
+		sort.Strings(ids)
+		sort.Strings(respIDs)
+		require.Equal(t, ids, respIDs)
```

Comment:
```suggestion
		respIDs := make([]string, 0, len(ids))
		for _, session := range respObj {
			respIDs = append(respIDs, session.ID)
		}
		require.ElementsMatch(t, ids, respIDs)
```

When we dont care about ordering the `ElementsMatch` function is the right one to use.